WindowsServerDocs/administration/windows-commands/nslookup.md

```diff
@@ -25,7 +25,7 @@ If you need to look up more than one piece of data, you can use interactive mode
 
 - Exit, by typing **exit**.
 
-- Treat a built-in command as a computer name, by preceding it with the escape character (\). An unrecognized command is interpreted as a computer name.
+- Treat a built-in command as a computer name, by preceding it with the escape character (\\). An unrecognized command is interpreted as a computer name.
```

Comment:
Good improvement, this update & correction is required. Could we also take the time to add 1 single blank space before & after the backslash(es) to make the combination of parentheses and backslash look less like an integrated symbol?
```suggestion
- Treat a built-in command as a computer name, by preceding it with the escape character ( \\ ). An unrecognized command is interpreted as a computer name.
```

Reply:
what about "(`\`)"? :)